webapp/src/ts/modules/messages/messages.component.ts

```diff
@@ -75,6 +55,42 @@ export class MessagesComponent implements OnInit, OnDestroy {
     this.subscriptions.unsubscribe();
   }
 
+  private reset() {
+    this.selectedConversationId = null;
```

Comment:
I thought of leaving this reset function with one line, in case we want to do more related to the reset then we already have a place for it instead of growing big in the subscriber's callback.